underscore

```diff
@@ -0,0 +1 @@
+Subproject commit 101c10a60019fe870d21868835f65c25d64968fc
```

Comment:
Hm why's a submodule necessary? Given that the package [implies](https://github.com/mixmaxhq/meteor-underscore-updates/blob/master/package.js#L12) Underscore.